Conceptpower+Spring/src/main/java/edu/asu/conceptpower/app/core/model/impl/ReviewRequest.java

```diff
@@ -0,0 +1,118 @@
+package edu.asu.conceptpower.app.core.model.impl;
+
+import java.io.Serializable;
+import java.time.OffsetDateTime;
+import java.util.List;
+
+import javax.persistence.CascadeType;
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.Id;
+import javax.persistence.OneToMany;
+import javax.persistence.Table;
+
+import edu.asu.conceptpower.core.ReviewStatus;
+
+/**
+ * This model helps to keep track of information about a specific review request
+ * 
+ * @author Keerthivasan Krishnamurthy
+ * 
+ */
+
+@Entity
+@Table(name="review_request")
+public class ReviewRequest implements Serializable{
+
+    private static final long serialVersionUID = 1L;
+
+    @Id
+    @Column(name="review_id")
+    private String id;
+    
+    @Column(name="concept_id")
+    private String conceptId;
+    
+    @Column(name="request_desc")
+    private String request;
+    
+    @Column(name="status")
+    private ReviewStatus status;
```

Comment:
you probably want to use the `Enumerated` annotation here and set it to use the string representation for the enum. I think the default is the index, which easily introduces bugs when whoever makes changes to the enum forgets and changes the order. With string values at least it's easy to spot when looking at the database.